Modules/Office365DSC/Modules/O365DSCReverse.psm1

```diff
@@ -42,8 +42,14 @@ function Start-O365ConfigurationExtract
     )
 
     $InformationPreference = "Continue"
-    $VerbosePreference = "SilentlyContinue"
-    $WarningPreference = "SilentlyContinue"
+
+    $DefaultWarningPreference = $WarningPreference
```

Comment:
what is the difference between commenting this out and this logic with Default...?

Reply:
This way default preferences will be those that we define before calling Start-O365ConfigurationExtract.

For example if we have verbose enabled and set it before calling the script, this code here will override it no matter what and we don't want that.